apps/web/src/server/api/routers/project.ts

```diff
@@ -32,4 +32,10 @@ export const projectRouter = createTRPCRouter({
     .mutation(async ({ ctx, input }) => {
       await ctx.db.insert(projects).values(input);
     }),
+
+  deleteProject: protectedProcedure
+    .input(z.object({ projId: z.number() }))
+    .mutation(async ({ ctx, input }) => {
+      await ctx.db.delete(projects).where(eq(projects.id, input.projId));
+    }),
```

Comment:
Change `projId` to just `id`. It's in the project router, we can get from context what id it's refering to.

Reply:
I didn't come with the naming convention of this crud procedures. I added delete resembling the other procedures.